### DISCUSSION TITLE
Does Zed support extensions?

### DISCUSSION THREAD

**Discussion by TomasHubelbauer** [zed-industries/zed#4250] [Feature Requests]
Hi, new to Zed, here from the open-sourcing announcement. I have looked at the website and I can't find an answer to whether Zed supports extensions and if so, what language can they be authored in. Thanks in advance for clarifying!

**Comment by gwhilts** [comment]
It doesn't appear that Zed supports user installed extensions at this point. 

From the FAQ:

> We plan to make Zed extensible. We won't add a plugin system before we open source Zed, but once that is complete, we will be working on extensibility. The exact details on what sort of languages and APIs we will support are still being worked out.

**Reply by dangdennis** [reply]
The [contribution docs](https://github.com/zed-industries/zed/blob/main/CONTRIBUTING.md) requests for work on this said extension feature. 

The plugin system is a top ranked issue already [link](https://github.com/zed-industries/zed/issues/5269)

Was hoping to add some lsp support too.

**Comment by JosephTLyons** [comment]
We have an extension store now that supports theme and language extensions.

command palette > `zed: extensions`:

<img width="1251" alt="SCR-20240708-pebb" src="https://github.com/zed-industries/zed/assets/19867440/7da987da-0d33-4d77-b20d-ad3b6621a968">

**Reply by harshgupta-mindit** [reply]
@JosephTLyons Can i made extensions for Zed if yes, where to find the docs for it, since i haven't found any content for creating extension for zed.

**Reply by SomeoneToIgnore** [reply]
Have you looked at https://github.com/zed-industries/extensions?tab=readme-ov-file#authoring-extensions ?

**Reply by harshgupta-mindit** [reply]
> Have you looked at https://github.com/zed-industries/extensions?tab=readme-ov-file#authoring-extensions ?

How did i miss it, Thanks @SomeoneToIgnore
